resources/timerbox.ts

```diff
@@ -431,9 +452,20 @@ export default class TimerBox extends HTMLElement {
       this._animationFrame = window.requestAnimationFrame(this.advance.bind(this));
     }
 
+    const remainingTime = Math.max(0, this._duration - elapsedSec);
+    if (remainingTime <= this._threshold && this._soundNotification === 'threshold' && this._playSoundNotification && this._duration > 0)
+      this.playNotification();
+
     this.draw();
   }
 
+  playNotification(): void {
+    this._playSoundNotification = false;
+    const audio = new Audio('../../resources/sounds/BigWigs/Alert.ogg');
+    audio.volume = 0.3;
```

Comment:
For now just hard-coded, should probably be configurable similar to the eureka overlay.

Reply:
I think it would be nice to have some new sound for this, to avoid making it sound like mechanics are happening.  Do you want to find an acceptably licensed sound off of freesound.org or something?)

Reply:
I'll look into it.